artifact-size-enforcer/src/main/java/org/codice/ddf/admin/beta/enforcer/ArtifactSizeEnforcerRule.java

```diff
@@ -0,0 +1,202 @@
+    public static final String MEGA_BYTES = "_MB";
+    public static final String KILO_BYTES = "_KB";
+
+    public static final String JAR = "jar";
+    public static final String BUNDLE = "bundle";
+    public static final List<String> SUPPORTED_PACKAGE_TYPES = Arrays.asList(JAR, BUNDLE);
+
+    //Rule arguments
+    private String maxArtifactSize;
+    private String artifactLocation;
+    private boolean skip;
+
+    private EnforcerRuleHelper helper;
+    private Log logger;
+
+    @Override
+    public void execute(EnforcerRuleHelper helper) throws EnforcerRuleException {
+        this.helper = helper;
+        this.logger = helper.getLog();
+
+        if (skip) {
+            return;
+        }
+
+        String packaging = getPackaging();
+        if (!SUPPORTED_PACKAGE_TYPES.contains(packaging)) {
+            logger.info(String.format(
+                    "Unsupported package type %s. Skipping artifact size enforcement.",
+                    packaging));
+            return;
+        }
+
+        String artifactPath = getArtifactPath();
+        long maxArtifactSizeBytes = maxArtifactSizeToBytes();
+        long artifactSize = getArtifact(artifactPath).length();
+
+        if (artifactSize >= maxArtifactSizeBytes) {
+            throw new EnforcerRuleException(String.format(
+                    "The specified artifact is larger than the set maximum artifact size. \n\n\tArtifact: %s\n\tArtifact Size: %s Bytes\n\tMax Artifact Size: %s Bytes\n"
+                            + "Either reduce the artifact size (Highly recommended) or set the maxArtifactSize property in the enforcer-plugin to a higher value.",
+                    artifactPath,
+                    artifactSize,
+                    maxArtifactSizeBytes));
+        }
+    }
+
+    private String getArtifactPath() throws EnforcerRuleException {
+        String convertedArtifactLocation = artifactLocation;
+
+        if (!StringUtils.isEmpty(convertedArtifactLocation)) {
+            logger.info(String.format("Using specified artifactLocation %s",
+                    convertedArtifactLocation));
+        } else {
+            try {
+                logger.warn(
+                        "artifactLocation property not specified. Looking up artifact using maven properties.");
+
+                String artifactId = (String) helper.evaluate(PROJECT_ARTIFACT_ID_PROP);
+                String version = (String) helper.evaluate(PROJECT_VERSION_PROP);
+                String packaging = getPackaging();
+                String buildDir = (String) helper.evaluate(PROJECT_BUILD_DIR_PROP);
+                logger.info(String.format(
+                        "Using the following parameters to find artifact: \n\tArtifactId: %s\n\tVersion: %s\n\tPackaging: %s\n\tArtifact Directory: %s",
+                        artifactId,
+                        version,
+                        packaging,
+                        buildDir));
+
+                convertedArtifactLocation =
```

Comment:
This is going to lead to misleading `INFO` logs. There are no artifacts generated with the file suffix of `bundle`.

Unless I'm misreading this code.